admin/onpage/class-onpage-notify.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Notify the admin there is an indexibility change
+ */
+class WPSEO_OnPage_Notify {
+
+	/**
+	 * Constructing the notify object
+	 *
+	 * @param int|null $old_status The old indexability status.
+	 * @param int      $new_status The new indexability status.
+	 */
+	public function __construct( $old_status, $new_status ) {
+		$this->send_email( $old_status, $new_status );
+		$this->show_notices();
```

Comment:
I really dislike side effects such as deleting database rows and sending emails from a constructor. I prefer a `notify` function because then it's explicit what's happening.

This would also mean the class could be renamed to `WPSEO_OnPage_Notification`.